static/styles/popovers.scss

```diff
@@ -86,6 +86,10 @@ ul.topics_popover i {
     margin-right: 3px;
 }
 
+ul.info_popover_actions i.fa-envelope {
+    font-size: .9em;
+}
```

Comment:
Let's use a number of pixels for this?  That's what we've been doing with other icons.